app/src/main/kotlin/com/simplemobiletools/camera/interfaces/MyPreview.kt

```diff
@@ -3,17 +3,18 @@ package com.simplemobiletools.camera.interfaces
 import android.net.Uri
 
 interface MyPreview {
-    fun onResumed()
 
-    fun onPaused()
+    fun onResumed() = Unit
```

Comment:
Provided default implementations for methods that are not used by the `CameraXPreview` so we don't have to override